example/css/example.scss

```diff
@@ -141,3 +141,20 @@ input {
     margin-top: 0;
   }
 }
+.singleDateRange{
```

Comment:
updated as requested. Here is screen shot as requested:
<img width="1056" alt="screen shot" src="https://cloud.githubusercontent.com/assets/3673943/13503299/b332025e-e13c-11e5-9593-b35f799f147b.png">